src/main/java/gregtech/loaders/recipe/chemistry/TitaniumRecipes.java

```diff
@@ -0,0 +1,157 @@
+                .buildAndRegister();
+    }
+
+    // The production of Soda Ash and Calcium Chloride from Salt and Calcite
+    // Used in the Bauxite Process
+    private static void solvayProcess() {
+        // CaCO3 -> CaO + CO2
+        CHEMICAL_RECIPES.recipeBuilder()
+                .circuitMeta(1)
+                .input(dust, Calcite, 5)
+                .output(dust, Quicklime, 2)
+                .fluidOutputs(CarbonDioxide.getFluid(1000))
+                .duration(200).EUt(VA[LV]).buildAndRegister();
+
+        // NaCl(H2O) + CO2 + NH3 -> NH4Cl + NaHCO3
+        CHEMICAL_RECIPES.recipeBuilder()
+                .input(dust, Salt, 4)
+                .fluidInputs(CarbonDioxide.getFluid(1000))
+                .fluidInputs(Ammonia.getFluid(1000))
+                .fluidInputs(Water.getFluid(1000))
+                .output(dust, AmmoniumChloride, 6)
+                .output(dust, SodiumBicarbonate, 6)
+                .duration(400).EUt(VA[MV]).buildAndRegister();
+
+        // 2NaHCO3 -> Na2CO3 + CO2 + H2O
+        ELECTROLYZER_RECIPES.recipeBuilder()
+                .input(dust, SodiumBicarbonate, 12)
+                .output(dust, SodaAsh, 6)
+                .fluidOutputs(CarbonDioxide.getFluid(1000))
+                .fluidOutputs(Water.getFluid(1000))
+                .duration(200).EUt(VA[MV]).buildAndRegister();
+
+        // 2NH4Cl + CaO -> CaCl2 + 2NH3 + H2O
+        CHEMICAL_RECIPES.recipeBuilder()
+                .input(dust, AmmoniumChloride, 12)
+                .input(dust, Quicklime, 2)
+                .output(dust, CalciumChloride, 3)
+                .fluidOutputs(Ammonia.getFluid(2000))
+                .fluidOutputs(Water.getFluid(1000))
+                .duration(200).EUt(VA[MV]).buildAndRegister();
+    }
+
+    // Advanced separation process for Bauxite
+    private static void bauxiteProcess() {
+        // Bauxite (crushed) + Soda Ash + Calcium Chloride -> Bauxite Slurry
+        MIXER_RECIPES.recipeBuilder()
+                .input(crushed, Bauxite, 32)
+                .input(dust, SodaAsh, 12)
+                .input(dust, CalciumChloride, 6)
+                .fluidInputs(Water.getFluid(4000))
+                .fluidOutputs(BauxiteSlurry.getFluid(4000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite (washed) + Soda Ash + Calcium Chloride -> Bauxite Slurry
+        MIXER_RECIPES.recipeBuilder()
+                .input(crushedPurified, Bauxite, 32)
+                .input(dust, SodaAsh, 12)
+                .input(dust, CalciumChloride, 6)
+                .fluidInputs(Water.getFluid(4000))
+                .fluidOutputs(BauxiteSlurry.getFluid(4000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite Slurry -> Cracked Bauxite Slurry
+        CRACKING_RECIPES.recipeBuilder()
+                .circuitMeta(1)
+                .fluidInputs(BauxiteSlurry.getFluid(16000))
```

Comment:
Should this be changed to do `4000 -> 4000` and have an appropriately reduced duration? Steam input can probably be left alone.

Reply:
I like the idea of having the recipes use different fluid amounts to affect overclocking/parallelization techniques a bit